app/views/hyrax/base/_form_share.html.erb

```diff
@@ -62,10 +62,21 @@
 <h2>Share work with groups of users</h2>
 <p class="sr-only">Use the add button to give access to one group at a time (it will be added to the list below).</p>
 <div class="form-group row">
+  <!--<div class="col-sm-5">-->
+    <!--<label for="new_group_name_skel" class="sr-only">Group</label>-->
+    <!--<%=# select_tag 'new_group_name_skel', options_for_select(["Select a group"] + current_user.groups), class: 'form-control' %>-->
+  <!--</div>-->
+
   <div class="col-sm-5">
+    <%
```

Comment:
What would you think of an alternative approach to exposing this logic in a view, like one of the following two:

1. Expose a new method -- on the form object or the user object? not sure where the best place for this is -- called something intention-revealing like `groups_for_sharing` that returns a list of groups a user belongs to with `'admin'` filtered out. This would also allow us to filter out other system-controlled groups such as `'registered'` and `'public'`. That way the logic lives somewhere reusable, outside of a view.
2. Alternatively, make this a one-line change:

```ruby
<%= select_tag 'new_group_name_skel', options_for_select(["Select a group"] + current_user.groups.without('admin'), class: 'form-control' %>
```

This latter approach improves upon the current code because you can safely call `.without('admin')` even if the current user if not an admin, and saves the cost of creating and mutating a new view-specific variable. The `if` there is pretty innocuous, I admit, but I'm remembering our Sandi Metz course and keeping in mind that branches/conditions increase complexity.